boot/core/src/boot/cli.clj

```diff
@@ -43,7 +43,17 @@
     code  (constantly true)
     file  #(instance? java.io.File %)))
 
-(defn- parse-fn [optarg]
+(defn- parse-fn
+  "Return a function accepting arguments that are split according to optarg.
+
+  The function works also for optargs splitting:
+
+    ((parse-fn SYM:VER=PATH) SYM:VER=PATH) => [SYM VER PATH]
+
+  But the typical usage is:
+
+    ((parse-fn SYM:VER=PATH) \"prj:1.3=parent\") => [prj 1.3 parent]"
+  [optarg]
```

Comment:
Would this also work?
```
((parse-fn SYM:VER=PATH=) \"prj:1.3=parent=xxx\")
```

Reply:
Probably no because the parsing needs `A-Z` interleaved by symbols at the moment

Reply:
Yes it works, I tried it by having a test, one second I am going to push it